EnglishFilter/sections/whitelist.txt

```diff
@@ -42,6 +42,10 @@ youtube.com#@##header.ytd-c4-tabbed-header-renderer:style(position: absolute !im
 ! https://github.com/AdguardTeam/PopupBlocker/issues/215
 @@||senpa.io^$extension
 @@||vanis.io^$extension
+! https://github.com/AdguardTeam/AdguardFilters/issues/70536
+inc42.com#@#.tag-social
+@@||piano.io^$domain=inc42.com
+@@||tinypass.com^$domain=inc42.com
```

Comment:
As far as I understand `inc42.com#@#.tag-social` is needed, because it hides content and it's from Fanboy's Social Blocking List/Fanboy's Annoyances.
What about these two:
```
@@||piano.io^$domain=inc42.com
@@||tinypass.com^$domain=inc42.com
```
As far as I see `tinypass.com` is blocked by `Fanboy's Enhanced Tracking List`, and it looks like that it breaks a website, so fine. But could you please tell me why `@@||piano.io^$domain=inc42.com` is needed?

Reply:
With piano.com^ blocked:
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/103278635-04ad5300-49dd-11eb-99e4-ee398051f576.png)
</details><br/>
And login button is disabled.

Reply:
Which filter list block `piano.com`?

Reply:
Peter Lowe's Blocklist

Reply:
Ok then, but I guess that we should also add to `Tracking Protection filter`:
`||piano.io/tracker/$domain=inc42.com,important`